udata/tests/forms/test_image_field.py

```diff
@@ -77,7 +77,7 @@ def test_with_image(self):
             doc.image.save(img, 'image.jpg')
         doc.save()
         form = self.F(None, obj=doc)
-        self.assertEqual(form.image.filename.data, 'image.jpg')
+        self.assertEqual(form.image.filename.data, 'image.png')
```

Comment:
It needs changes at the flask-fs level. I can try to keep the original name and add `.optimized` to the new rendered file.